frontend/pages/queries/QueryPage/QueryPage.jsx

```diff
@@ -567,24 +568,25 @@ const mapStateToProps = (state, ownProps) => {
   const queryStub = { description: '', name: '', query: queryText };
   const query = reduxQuery || queryStub;
   const { selectedTargets } = state.components.QueryPages;
-  const { host_ids: hostIDs } = ownProps.location.query;
+  const { host_ids: hostIDs, host_uuids: hostUUIDs } = ownProps.location.query;
   const { isSmallNav } = state.app;
   const title = queryID ? 'Edit Query' : 'New Query';
   let selectedHosts = [];
 
-  // hostIDs are URL params so they are strings
-  if (hostIDs && !queryID) {
-    const hostFilter = isArray(hostIDs)
-      ? h => includes(hostIDs, String(h.id))
-      : { id: Number(hostIDs) };
-
+  if (!queryID && (hostIDs || hostUUIDs)) {
+    const hostIDsArr = Array.isArray(hostIDs) ? hostIDs : [hostIDs];
+    const hostUUIDsArr = Array.isArray(hostUUIDs) ? hostUUIDs : [hostUUIDs];
     const { entities: hosts } = stateEntities.get('hosts');
+    // hostIDs are URL params so they are strings and comparison with ints may
+    // need conversion.
+    const hostFilter = h => includes(hostIDsArr, String(h.id)) || includes(hostUUIDsArr, String(h.uuid));
```

Comment:
If you're replacing lodash functionality with native for `isArray`, we could do the same for [`includes`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes)

Reply:
What do you think the tradeoffs are? Should I always prefer lodash? I see that `isArray` is compatible with every browser on MDN, while `includes` is compatible with all besides IE.

Reply:
It's being compiled with Babel, so we can make sure it supports the required browsers - https://babeljs.io/docs/en/babel-preset-env#targets

Reply:
Nice, we have the browsers specified in https://github.com/kolide/fleet/blob/master/package.json#L118-L124, so I think we're good here?

Reply:
It's really up to you. Maybe it's just a followup PR where we reassess our uses of lodash?

Reply:
To finalize this, Kyle and I connected over chat and we believe that the Babel configuration will handle appropriate browser support.